node/silkworm/common/directories_test.cpp

```diff
@@ -23,18 +23,6 @@
 namespace silkworm {
 
 TEST_CASE("DataDirectory") {
```

Comment:
I would not remove the test rather initialize it from a temporary directory.
We get the same result not to tamper with default storage path and keep the test
```suggestion
TEST_CASE("DataDirectory") {
    {
        // Open and create a storage path
        TemporaryDirectory tmp_dir0;
        DataDirectory data_dir(/*base_path=*/tmp_dir0.path(), /*create=*/true);
        REQUIRE(data_dir.exists());
        REQUIRE_NOTHROW(data_dir.deploy());
        REQUIRE_THROWS(data_dir.clear());

        // Eventually delete the created paths
        std::filesystem::remove_all(data_dir.path());
        REQUIRE(data_dir.exists() == false);
    }

```

Reply:
Or ... another option is to keep the test as it is and continue the test only if `data_dir.is_pristine() == true` (which means there is nothing in it). In such case the test for initialization from default storage path succeeds while subsequent deletions get skipped.

Reply:
I've switched to your temporary directory suggestion just to be on the safe side and avoid doing anything in potentially sensitive directories.